editoast/src/views/timetable.rs

```diff
@@ -1,3 +1,4 @@
+pub mod occupancy_block;
```

Comment:
```suggestion
mod occupancy_block;
```

does it need to be pub?

Reply:
still needed in `projection.rs` for the use of `compute_batch_signal_update`, I added a TODO to avoid it in the next PR, I can add another TODO here to be sure to remove it too if you want